### ISSUE TITLE
EMS registry number should be a number

### ISSUE THREAD

**Issue by o-fish-wildaid** [WildAid/o-fish-android#272]
Please show the numeric keyboard as default for entering information into the EMS section - registry number.

**Comment by subbramanil** [comment]
I would like to take a stab at it in the Hacktoberfest. Could you please assign it for me @o-fish-wildaid.

**Comment by am-MongoDB** [comment]
@ subbramanil Welcome to the O-FISH project! I've assigned this issue to you

**Comment by subbramanil** [comment]
Changing the input type to 'number' helps android to use the appropriate keyboard. 

To be honest, I sort of feel bad about submitting a PR with one line change. I understand that during Hacktoberfest, I wouldn't be assigned more than one issue to work. However, I would like to contribute more once the Hacktoberfest is over.

**Comment by Sheeri** [comment]
Hi @subbramanil  - I would like to clarify - You may be assigned more than one issue DURING hacktoberfest, just not at the same time.

For example, you could work on issue 12345 and then 12346. However, you cannot be assigned both at the same time. This is to avoid people 'claiming' a few tickets and then not working on them. In tech terms, you can work on multiple issues in serial, but not in parallel.

If there's another issue you'd like to work on, let us know and we'll assign it to you!

**Comment by subbramanil** [comment]
@Sheeri It makes a lot of sense now. I thought earlier the rule was set to give a chance to someone else to contribute during Hacktoberfest.

I am working on a couple of issues on a few other repositories. I will come back and check out the list of issues, once I finish them. 

Thank you for the clarification and opportunity to contribute more.
